### ISSUE TITLE
bibxml7 issue

### ISSUE THREAD

**Issue by kesara** [ietf-tools/bibxml-service#313]
### Describe the issue

Example: https://bib.ietf.org/get-one/by-docid/?doctype=IETF&docid=RFC+9000&format=bibxml

Clicking on the second BibXML reference link (https://bib.ietf.org/public/rfc/bibxml7/reference.DOI.10.17487/RFC9000.xml) generates the following error:

```
{"error": {"message": "Error resolving bibliographic item. Tried methods: auto (DoiAdapter: ): validation problem, fallback (): not indexed"}}
```

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by ronaldtse** [comment]
Thanks @kesara : this looks like an issue with fetching a DOI with a slash?

**Comment by strogonoff** [comment]
@kesara I believe this may have to do with a regression regarding to fetching DOI, and which should be addressed in the latest PR. This issue affected certain DOIs (e.g., those that specified contributor roles), and also could cause an error 500 when using DOI lookup widget on landing page.

Staging link works now: https://dev.bibxml.org/public/rfc/bibxml7/reference.DOI.10.17487/RFC9000.xml

**Comment by kesara** [comment]
Thanks, @strogonoff, fix has been deployed.
Fixed in #314.
